File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/BootService.java
package com.example.sarahn.waterreminderapp.classes;

import android.app.IntentService;
import android.content.Intent;

import com.example.sarahn.waterreminderapp.Utils.AlarmManagerUtils;
import com.example.sarahn.waterreminderapp.Utils.Logging;

import java.util.Date;

/**
 * Created by SarahN on 7/3/2017.
 */
public class BootService extends IntentService {

    public BootService() {
        super("boot service");
    }

    @Override
    protected void onHandleIntent(Intent intent) {
        AlarmManagerUtils.setStartAlarm();
        AlarmManagerUtils.setEndAlarm();
    }
}

File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/BootService.kt
package com.example.sarahn.waterreminderapp.classes

import android.app.IntentService
import android.content.Intent
import com.example.sarahn.waterreminderapp.Utils.AlarmManagerUtils

/**
 * Created by SarahN on 7/3/2017.
 */
class BootService : IntentService("boot service") {
    override fun onHandleIntent(intent: Intent?) {
        AlarmManagerUtils.setStartAlarm()
        AlarmManagerUtils.setEndAlarm()
    }
}